services/app-api/forms/2026/elements.ts

```diff
@@ -156,6 +165,8 @@ export const didYouFollowSpecifications: RadioTemplate = {
           id: "measure-following-tech-specs-no-explain",
           label: "Please explain the variance.",
           required: true,
+          helperText:
+            "Include the name of which technical specifications that were used in the reporting of this measure, or any data elements that were collected outside of the most current guidance (e.g. sampling size, population, denomination calculation etc.)",
```

Comment:
This does match the wording in the document, but I think the wording in the document is probably incorrect, because it's ungrammatical. Here's one way we might fix it:

```suggestion
          helperText:
            "Include the name of which technical specifications were used in the reporting of this measure, or any data elements that were collected outside of the most current guidance (e.g. sampling size, population, denomination calculation etc.)",
```